feat: quick filters

daibhin opened the pull request:
## Problem

We ran an experiment on quick filters and think that this design performs as well for folks and should make the most common use cases easier

## Changes

- Remove experiment feature flag
- Add scrolling to filters
- Add prompt to add person properties
- Fix scrolling in settings pages

<img width="325" alt="Screenshot 2023-08-17 at 14 02 24" src="https://github.com/PostHog/posthog/assets/6685876/4ef69b00-75ee-4323-a726-5c09a6c2137c">

## How did you test this code?

By 👁️

Commit:
add scrolling & person property suggestions

Commit:
update explaination

Commit:
remove feature flag

Commit:
fix settings scrolling

github-actions[bot] committed:
Update UI snapshots for `chromium` (1)

github-actions[bot] committed:
Update UI snapshots for `chromium` (2)

xvello committed:
chore(plugin-server): add tags to redisGet/Set (#17057)

Commit:
feat: Use buildjet for ci-backend (#17069)

* ci-backend runs-on -> buildkite

* Use buildjet runners

* Go away old cache

* Fix deps

* Ensure to update the package registry

* Use bigger runners for django tests

* Use ARM for jobs with less concurrency

* Revert "Use ARM for jobs with less concurrency"

This reverts commit bcbf0d752412d4231b7b86c3413022b1d12facb4.

* Fix typo

tiina303 committed:
chore: Allow impersonated users to trigger historical exports (#16979)

pauldambra commented on frontend/src/scenes/session-recordings/filters/SimpleSessionRecordingsFilters.tsx:
```diff
@@ -82,41 +85,50 @@ export const SimpleSessionRecordingsFilters = ({
+                {displayNameProperties.length === 0 && (
+                    <Link to={urls.projectSettings('person-display-name')}>
+                        <LemonButton type="tertiary" size="small" className="whitespace-nowrap" icon={<IconPlus />}>
+                            Add person properties
```
💯

tiina303 committed:
chore: metrics for step completions within webhooks (#17056)

Commit:
fix: Docker compose not picking up env for some reason (#17079)

Docker compose not picking up env for some reason

Will this help?

pauldambra commented on frontend/src/lib/hooks/useAnchor.ts:
```diff
@@ -11,11 +11,8 @@ export function useAnchor(hash: string): void {
-                top: elementPosition - 50 - 32, // compensate for header & top margin of pages
-                behavior: 'smooth',
-            })
+
```
noticed this only works for me the second time I click the link

(or rather not the first time after a refresh)

![2023-08-17 17 45 10](https://github.com/PostHog/posthog/assets/984817/99e56867-0c1d-4a1e-af9c-1d79e38cf4e0)

daibhin replied:
Fixed with a `window.requestAnimationFrame`

pauldambra commented on frontend/src/scenes/session-recordings/filters/SimpleSessionRecordingsFilters.tsx:
```diff
@@ -47,7 +50,7 @@ export const SimpleSessionRecordingsFilters = ({
     return (
         <div className="space-y-2">
-            <div className="flex space-x-1">
+            <div className="flex space-x-1 px-3 overflow-x-scroll pb-3">
```
should this have `overflow-x-scroll` or should it wrap?

![2023-08-17 17 45 10](https://github.com/PostHog/posthog/assets/984817/43543bfb-cb7e-482e-8541-1de2608ff070)

daibhin replied:
Great shout! Wrapping is much nicer

raquelmsmith committed:
feat: sync distinct ids with billing (#16919)

* update billing distinct IDs on user join/leave org

* temporarily update on usage report send
to update existing orgs

* abstract a bit more

* update existing tests

* add billing_manager test for the method

* Update query snapshots

* specify date

* move outside the transaction

* cache the instance license

* use new instance license method in other places

* fix mypy and some tests

* Update query snapshots

* Update query snapshots

* fix caching in tests

* constrain try/except

* Update query snapshots

* make sure we hae license before update billing

* Update query snapshots

* Update query snapshots

* set to false if no instance license

* fix date

* Update query snapshots

* use correct var

* Update UI snapshots for `chromium` (1)

* clear license cache before running tests

* Update UI snapshots for `chromium` (2)

* Update UI snapshots for `chromium` (2)

* fix tests

---------

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

liyiy committed:
feat(surveys): exclude users who have already seen survey wait period (#17064)

* add condition value for wait period

* do not allow for wait periods under 1 day

* Update UI snapshots for `chromium` (1)

* Update UI snapshots for `chromium` (2)

---------

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

posthog-bot committed:
chore(deps): Update posthog-js to 1.77.0 (#17085)

Twixes committed:
fix(insights): Hide empty "Display" options section (#17044)

* fix(insights): Hide empty "Display" options section

* Don't use different color for options count

pauldambra committed:
fix: group notifications by five minutes (#17077)

* fix: group notifications by minute

* query even less datA

* fewer queries and a five minute trunc

* with attribution

xvello committed:
fix(exports): hopefully fix onevent stalling by catching job producing errors (#17087)

pauldambra committed:
feat: notebook filtering from notebooks table view (#17027)

* feat: filter notebooks in the notebooks table

* don't break other uses of the logic

* Comment

* Update frontend/src/scenes/notebooks/NotebooksList/NotebooksList.tsx

Co-authored-by: Ben White <ben@posthog.com>

* Update UI snapshots for `chromium` (2)

* wip

* fix

* Fix

* Update UI snapshots for `chromium` (2)

* more fangling

* Update UI snapshots for `chromium` (1)

* Update UI snapshots for `chromium` (2)

* Update UI snapshots for `chromium` (2)

* fix

* different labels

* Update UI snapshots for `chromium` (2)

* obey mypy

* setup backend for filtering as text

* setup backend for filtering as text

* ah but

* fix

* Update UI snapshots for `chromium` (1)

* move title search to backend

---------

Co-authored-by: Ben White <ben@posthog.com>
Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

Twixes committed:
fix(insights): Show group analytics in exported/shared insights (#17041)

* fix(insights): Show group analytics in exported/shared insights

* Also fix empty check mark

* Mock group types response in stories

* Update test assertions

* Also update frontend logic tests

* Update UI snapshots for `chromium` (2)

* Update UI snapshots for `chromium` (2)

* Update propertyDefinitionsTableLogic.test.ts

* Fix an issue in group types viewset and add tests for good measure

More context here: https://posthog.slack.com/archives/C0428JHABK2/p1692361040699229

* Update query snapshots

* Update UI snapshots for `chromium` (2)

---------

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

pauldambra committed:
chore: jest on buildjet (#17095)

* chore: jest on buildjet

* try this

* maybe 8:

* no 4

pauldambra committed:
chore: cypress on buildjet (#17096)

* cypress buldjet maybe

* like this?

* try it on a matrix

* on 4 cpu

* typo

pauldambra committed:
chore: fix story date (#17097)

* chore: fix story date

* Update UI snapshots for `chromium` (1)

---------

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

raquelmsmith committed:
feat: add data model and initial screen for product-specific onboarding (#17081)

* add products scene

* hover effects on ProductCard

* support dynamic onboarding completed vals

* add onboarding json field to team

* increase width, center text

* add a feature flag

* Update query snapshots

* Update query snapshots

* Update UI snapshots for `chromium` (1)

* Update query snapshots

* make the field show up in the org response

* Update UI snapshots for `chromium` (1)

* Update UI snapshots for `chromium` (1)

* Update UI snapshots for `chromium` (1)

---------

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

pauldambra committed:
chore: use buildjet for visual regression tests (#17098)

* like this?

* run if the workflow changed

* not 2 cpu

* 4 is enough oomph

mariusandra committed:
chore(storybook): upgrade storybook to 7.3 (#17078)

* feat(kea): typegen path cleanup and delete unused files

* bump

* 3.2.2

* typegen upgrade to 3.3.0

* storybook 7.3

* upgrade storybook to 7.3

* try using #storybook-root instead of #root

* try using #storybook-root instead of #root (2)

* try using #storybook-root instead of #root (3)

* try using #storybook-root instead of #root (4)

* switch out argTypes.defaultValue with args

* Update UI snapshots for `chromium` (1)

* Update UI snapshots for `chromium` (2)

* Update UI snapshots for `chromium` (1)

* Update UI snapshots for `chromium` (1)

---------

Co-authored-by: Thomas Obermüller <thomas.obermueller@gmail.com>
Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

Commit:
move to wrapping properties

Commit:
add repaint delay to anchor scrolling

Commit:
fix: Default Replay list duration (#17090)

Commit:
Merge branch 'master' into dn/quick-filters

github-actions[bot] committed:
Update UI snapshots for `chromium` (1)

github-actions[bot] committed:
Update UI snapshots for `chromium` (2)

github-actions[bot] committed:
Update UI snapshots for `chromium` (1)
